src/libxtp/gwbse/gw.cc

```diff
@@ -48,6 +49,7 @@ void GW::configure(const options& opt) {
   _sigma->configure(sigma_opt);
   _Sigma_x = Eigen::MatrixXd::Zero(_qptotal, _qptotal);
   _Sigma_c = Eigen::MatrixXd::Zero(_qptotal, _qptotal);
+  _mixing_order = _opt.gw_mixing_order;
```

Comment:
I do not think this is necessary you have value in the _opt struct.

Reply:
True.